src/views/Home.vue

```diff
@@ -108,9 +81,19 @@ export default {
     if (this.$route.query.settingsSuccess === 'true') {
       this.settingsSuccessMessage = true
     }
+    this.getPrices()
     this.getTodayPriceCount()
   },
   methods: {
+    getPrices() {
+      this.loading = true
+      return api.getPrices({ size: 5 })
```

Comment:
How many prices should we display ? 
Maybe load 5, but display only 2 or 3 on small screens ?

Reply:
I'd load up to 25, adding "3 minutes ago", up to"24 hours ago" instead of Today's date

Reply:
ok will re-add the "relative time info" in the PriceCard in a different PR

Reply:
and I need to think about loading more data for bigger screens (different PR as well)

Reply:
bumped to 25 (and kept it to 5 on small screens) in this commit : https://github.com/openfoodfacts/open-prices-frontend/commit/588c97a9f6ed14fe73bd39b0f0d5351973758071

Reply:
> instead of Today's date

its the price's date that is displayed. sometimes prices are added for a different date.
indeed it can be confusing..

Reply:
It was more to make it more alive and exciting, I wasn't particularly confused by anything